packages/types/src/interfaces/contracts/definitions.ts

```diff
@@ -239,7 +245,7 @@ export default {
       value: 'Balance',
       gasLimit: 'Gas',
       storageDepositLimit: 'Option<Balance>',
-      code: 'Bytes',
+      code: 'CodeSource',
```

Comment:
Also needs a previous version here.

Reply:
I believe it was always supposed to be an enum, and Bytes was just a placeholder? Anyway, it seems like enum definitions can't have fallbacks at the moment, can they? I tried and it just made a struct with _enum and _fallback fields